chrome/application-services-navigation.json

```diff
@@ -9,20 +9,57 @@
             "href": "/application-services/overview"
         },
         {
-            "title": "Streams for Apache Kafka",
+            "appId": "applicationServices",
+            "title": "API Management",
+            "href": "/application-services/api-management"
+        },
+        {
+            "appId": "applicationServices",
+            "title": "Connectors",
+            "href": "/application-services/cos"
+        },
+        {
+            "appId": "applicationServices",
+            "title": "Data Science",
+            "href": "/application-services/data-science"
+        },
+        {
+            "appId": "applicationServices",
+            "title": "Service Accounts",
+            "href": "/application-services/service-accounts"
+        },
+        {
+            "title": "Service Registry",
             "expandable": true,
             "routes": [
                 {
                     "appId": "applicationServices",
-                    "title": "Kafka Instances",
+                    "title": "Service Registry Instances",
                     "isBeta": true,
-                    "href": "/application-services/streams/kafkas"
+                    "href": "/application-services/sr"
```

Comment:
I think the desired URL is `/application-services/service-registry`

Reply:
I will change it.